sbag/language/library.sbag

```diff
@@ -49,6 +49,15 @@ entity Address:
     ;
 ;
 /custom
+    get: Book
+    post(Book): Publisher
+    put(Author): Author
+;
+/custom2
+    get: String
+    post(Book): String
+;
+/custom3
```

Comment:
Can this ones be renamed not to include numbers in their names. Something like /stuff, /points, /anything, /whatever

Reply:
They will be renamed.